.eslintrc

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": [ "apostrophe" ]
+  "extends": [ "apostrophe" ],
+  "rules": {
+    "no-var": 2
```

Comment:
Interesting, is this not standard in "apostrophe"?

Reply:
We didn't have it for the sake of A2 front end JS. We can probably do a major version bump on the config with it for A3. Override that the opposite way for old projects where needed.